lib/process-services-cloud/src/lib/people/components/people-cloud.component.spec.ts

```diff
@@ -824,6 +820,86 @@ describe('PeopleCloudComponent', () => {
         });
     });
 
+    describe('Groups restriction', () => {
+
+        beforeEach(() => {
+            fixture.detectChanges();
+            element = fixture.nativeElement;
+            findUsersByNameSpy = spyOn(identityService, 'findUsersByName').and.returnValue(of(mockUsers));
+        });
+
+        it('Shoud display all users if groups restriction is empty', async () => {
+            getInvolvedGroupsSpy = spyOn(identityService, 'getInvolvedGroups').and.returnValue(of(mockInvolvedGroups));
+            component.groupsRestriction = [];
+            fixture.detectChanges();
+            typeInputValue('M');
+            fixture.detectChanges();
+
+            await fixture.whenStable();
+            fixture.detectChanges();
```

Comment:
fixture.detectChanges();
await fixture.whenStable();
fixture.detectChanges();

looks suspicious why do we need 2 times fixture.detectChanges(); ?

Reply:
good point, removed